pkg/minishift/docker/image/image.go

```diff
@@ -27,10 +27,8 @@ import (
 type ImageMissStrategy int
 
 const (
-	SKIP ImageMissStrategy = iota
-	PULL
-	// TODO Implement a retry strategy as well (HF)
-	//RETRY
+	Skip ImageMissStrategy = iota
+	Pull
```

Comment:
👍 makes more sense to not follow the uppercase way of writing constants, and also no need for TODOs.